indigo_app/views/tasks.py

```diff
@@ -90,6 +105,8 @@ def get_context_data(self, **kwargs):
         context['total_tasks'] = self.get_base_queryset().count()
 
         context["taxonomy_toc"] = TaxonomyTopic.get_toc_tree(self.request.GET)
+        context["work_facets"] = self.form.work_facets(self.get_base_queryset(), context['taxonomy_toc'], [])
```

Comment:
In the place work view, we don't calculate facets on the main page of the page. Instead, we render a placeholder for the facet list and then load the facets via htmx. This means we keep that page fast to load.